tests/unit/Codeception/Coverage/FilterTest.php

```diff
@@ -95,12 +95,9 @@ public function testWhitelistIncludeFilterApplied()
 
     private function getFilterMethod(): string
     {
-        $filterMethod = 'isFiltered';
         if (method_exists($this->filter->getFilter(), 'isExcluded')) {
-            //php-code-coverage 9+
-            $filterMethod = 'isExcluded';
```

Comment:
`getFilterMethod` wrapper can be eliminated, 
 php-code-coverage 9.0 is used by PHPUnit ^9.3 and Codeception 5 has never supported anything older than 9.5.

Reply:
Done.